pkg/oc/cli/config/loader.go

```diff
@@ -8,15 +8,15 @@ import (
 )
 
 func NewPathOptions(cmd *cobra.Command) *kclientcmd.PathOptions {
-	return NewPathOptionsWithConfig(kcmdutil.GetFlagString(cmd, kclientcmd.OpenShiftKubeConfigFlagName))
+	return NewPathOptionsWithConfig(kcmdutil.GetFlagString(cmd, kclientcmd.RecommendedConfigPathFlag))
 }
 
 func NewPathOptionsWithConfig(configPath string) *kclientcmd.PathOptions {
 	return &kclientcmd.PathOptions{
```

Comment:
I was expecting a patch in this type that allowed the specification of two explicit flags.

Reply:
Done.